RecoTracker/TkHitPairs/src/HitPairGeneratorFromLayerPair.cc

```diff
@@ -86,26 +86,18 @@ void HitPairGeneratorFromLayerPair::hitPairs(
   }
 }
 
-
-HitDoublets HitPairGeneratorFromLayerPair::doublets( const TrackingRegion& region,
-						    const edm::Event & iEvent, const edm::EventSetup& iSetup, Layers layers) {
-
-  typedef OrderedHitPair::InnerRecHit InnerHit;
-  typedef OrderedHitPair::OuterRecHit OuterHit;
+void HitPairGeneratorFromLayerPair::doublets2(const TrackingRegion& region,
```

Comment:
I had the same reaction at first.
In reality most of the content of old doublets is now in doublets2, so this comparison is not really bad
(doublets now calls doublets2 at its very end...)